### PR TITLE
i18n(es): Update `customization`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updated `customization`

### CODE REVIEW

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/guides/customization.mdx [hunk 1]
```diff
@@ -168,7 +168,7 @@ tableOfContents: false
   </TabItem>
   <TabItem label="Configuración global">
 
-```js
+```js{7}
```
Comment (delucis) [inline docs/src/content/docs/es/guides/customization.mdx:171]: Oops, looks like the missing space here is causing the build to fail:
````suggestion
```js {7}
````

**Comment by dreyfus92** [comment]
Thank you so much @delucis & @Waxer59 for the review 😁

**Review by delucis — APPROVED** [review]
Looks good now — thank you! 🚀

**Pull request merged**